notebooks/model_training.ipynb

```diff
@@ -111,28 +107,41 @@
     "        if wandb_enabled:\n",
     "            wandb.log({\"validation/recall\": metrics.recall, \"validation/accuracy\": metrics.accuracy, \"validation/precision\": metrics.precision, \"validation/f1\": metrics.f1, \"epoch\": epoch+1})\n",
     "\n",
-    "        if metrics.f1 < best_f1:\n",
-    "            patience = patience + 1\n",
-    "        else:\n",
-    "            patience = 0\n",
-    "            best_f1 = metrics.f1\n",
-    "        if patience >= PATIENCE_THRESHOLD:\n",
```

Comment:
Didn't you remove saving best model instead of last?

Reply:
It is great that you spotted it. I haven't seen your code in my editor, so it must be git issue. I will fix it

Reply:
I have fixed it